src/main/java/org/aerogear/kryptowire/workflow/KWSubmitStep.java

```diff
@@ -77,11 +80,18 @@ public String getDisplayName() {
         @StepContextParameter
         transient TaskListener listener;
 
+        @StepContextParameter
+        private transient Run build;
+
         @Override
         protected Void run() throws Exception {
 
             GlobalConfigurationImpl pluginConfig = GlobalConfiguration.all().get(GlobalConfigurationImpl.class);
 
+            if (pluginConfig == null) {
+                throw new RuntimeException("[Error] Could not retrieve global Kryptowire config object.");
```

Comment:
If we are prepending "[Error]" here, we should probably do it in the RuntimeExcpetion we are throwing below. When would this ever be null?